probe/kubernetes/reporter.go

```diff
@@ -230,23 +230,23 @@ func (r *Reporter) Tag(rpt report.Report) (report.Report, error) {
 // Report generates a Report containing Container and ContainerImage topologies
 func (r *Reporter) Report() (report.Report, error) {
 	result := report.MakeReport()
-	serviceTopology, services, err := r.serviceTopology()
+	serviceTopology, services, err := r.serviceTopology(r.probeID)
 	if err != nil {
 		return result, err
 	}
 	hostTopology := r.hostTopology(services)
 	if err != nil {
 		return result, err
 	}
-	daemonSetTopology, daemonSets, err := r.daemonSetTopology()
+	daemonSetTopology, daemonSets, err := r.daemonSetTopology(r.probeID)
 	if err != nil {
 		return result, err
 	}
-	statefulSetTopology, statefulSets, err := r.statefulSetTopology()
+	statefulSetTopology, statefulSets, err := r.statefulSetTopology(r.probeID)
 	if err != nil {
 		return result, err
 	}
-	cronJobTopology, cronJobs, err := r.cronJobTopology()
+	cronJobTopology, cronJobs, err := r.cronJobTopology(r.probeID)
```

Comment:
These functions do not need to take the `r.probeID` since they are methods on `Reporter` and hence have access to it.